### PR TITLE
Changed the image versions

### PR DESCRIPTION
**Pull request by DipayanP007**
# Description

Changed the image version in the `values.yaml` for fluentd, thanos, rbacManager, kubePrometheusstack

# Issues

Solved the issue
* #526 
* #527 
* #528 
* #530  
# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

**Comment by hairmare** [comment]
CI indicates the `version` needs bumpig in both `charts/infra-apps/Chart.yaml` and `charts/loggig-apps/Chart.yaml`. The corresponding READMEs should also be updated to match the versions from Chart.yaml as well as the new targetRevisions.

File: charts/infra-apps/values.yaml [hunk 1]
```diff
@@ -116,7 +116,7 @@ kubePrometheusStack:
   # kubePrometheusStack.chart -- Chart
   chart: kube-prometheus-stack
   # kubePrometheusStack.targetRevision -- [kube-prometheus-stack Helm chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) version
-  targetRevision: 30.0.*
+  targetRevision: 30.1.0
```
Comment (hairmare) [inline charts/infra-apps/values.yaml:119]:
```suggestion
  targetRevision: 30.1.*
```

File: charts/infra-apps/values.yaml [hunk 2]
```diff
@@ -136,7 +136,7 @@ rbacManager:
   # rbacManager.chart -- Chart
   chart: rbac-manager
   # rbacManager.targetRevision -- [rbac-manager operator Helm chart](https://github.com/FairwindsOps/charts/tree/master/stable/rbac-manager) version
-  targetRevision: 1.9.*
+  targetRevision: 1.10.0
```
Comment (hairmare) [inline charts/infra-apps/values.yaml:139]:
```suggestion
  targetRevision: 1.10.*
```

File: charts/infra-apps/values.yaml [hunk 3]
```diff
@@ -156,7 +156,7 @@ thanos:
   # thanos.chart -- Chart
   chart: thanos
   # thanos.targetRevision -- [Thanos Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/thanos)
-  targetRevision: 8.2.*
+  targetRevision: 9.0.0
```
Comment (hairmare) [inline charts/infra-apps/values.yaml:159]:
```suggestion
  targetRevision: 9.0.*
```

File: charts/logging-apps/values.yaml [hunk 4]
```diff
@@ -53,7 +53,7 @@ fluentd:
   # fluentd.chart -- Chart
   chart: "fluentd"
   # fluentd.targetRevision -- [fluentd Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/fluentd) version
-  targetRevision: "4.4.*"
+  targetRevision: "5.0.0"
```
Comment (hairmare) [inline charts/logging-apps/values.yaml:56]:
```suggestion
  targetRevision: "5.0.*"
```

**Review by hairmare — DISMISSED** [review]
We currently pin versions with a wildcard char (`*`) for the patch version. This might change in the future but for now the targetRevisions should reflect that.

**Comment by DipayanP007** [comment]
Hey @hairmare about the Chart version. Should I change the patch version ?

**Comment by hairmare** [comment]
@DipayanP007 please change the minor version.

**Comment by DipayanP007** [comment]
Hey @hairmare I did the changes

**Review by eyenx — APPROVED** [review]
@DipayanP007 Thank you for your contribution!

**Pull request merged**